.github/workflows/ci.yml

```diff
@@ -47,13 +63,5 @@ jobs:
           # WORKDIR=/share docker run --rm -v $(pwd):$WORKDIR -w $WORKDIR --entrypoint slither trailofbits/eth-security-toolbox [insert slither parameters here]
 
       - name: Test
-        run: yarn test
+        run: npx hardhat test
 
-      - name: Test coverage
```

Comment:
Sad to see the coverage reports removed…. 😭

Reply:
I don't know if/how well it works in the CI (or uploading to/with codecov), but hardhat does have a coverage command that displays coverage information in the terminal and creates html pages in ./coverage:
```bash
yarn hardhat coverage
ls ./coverage
# base.css  contracts  coverage-final.json  index.html  lcov.info  lcov-report  prettify.css  prettify.js  sort-arrow-sprite.png  sorter.js
```

Reply:
that's how it previously worked (using the truffle equivalent). i guess it's not super important to upload, but tracking coverage for the contracts is a good practice and at least having a command for that in the package.json would be good, if uploading is too much trouble.